foundation/foundation-archetype/src/main/resources/archetype-resources/__rootArtifactId__-docker/pom.xml

```diff
@@ -86,16 +86,21 @@
             <build>
                 <plugins>
                     <plugin>
-                        <groupId>org.technologybrewery.orphedomos</groupId>
-                        <artifactId>orphedomos-maven-plugin</artifactId>
-                        <configuration>
-                            <skip>false</skip>
-                            <imageVersion>${project.version}</imageVersion>
-                            <!-- Add in repoId that corresponds to the serverId in your settings.xml
-                                 that contains the docker login credentials -->
-                            <repoId />
-                            <repoUrl>${docker.project.repository.url}</repoUrl>
-                        </configuration>
+                        <groupId>io.fabric8</groupId>
+                        <artifactId>docker-maven-plugin</artifactId>
+                        <executions>
+                            <execution>
+                                <id>default-build</id>
+                                <configuration>
+                                    <skip>false</skip>
+                                    <images>
+                                        <image>
+                                            <name>${docker.project.repository.url}/${project.artifactId}:${project.version}</name>
+                                        </image>
+                                    </images>
+                                </configuration>
+                            </execution>
+                        </executions>
```

Comment:
Discussed offline, but going to move these configs out of the `execution` block so that it will apply to all executions.